index.js

```diff
@@ -34,6 +34,7 @@ export const Rels = {
 	root: 'https://api.brightspace.com/rels/root',
 	richTextEditorConfig: 'https://api.brightspace.com/rels/richtext-editor-config',
 	rubric: 'https://api.brightspace.com/rels/rubric',
+	rubrics: 'https://api.brightspace.com/rels/rubrics',
```

Comment:
Aw - this one isn't namespaced onto rubrics.api.brightspace.com? 😢

Reply:
nope, unfortunately not 😢